helm/korifi/values.yaml

```diff
@@ -44,6 +44,14 @@ api:
       idle: 900
       readHeader: 10
 
+  infoConfig:
+    description: ""
+    name: ""
```

Comment:
Maybe we should default `name` and `description` so that it looks more similar to current CF systems. Something generic like:

```
name: "korifi"
description: "Korifi Cloud Foundry Environment"
```